README-PL.md

```diff
@@ -0,0 +1,144 @@
+
+[![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
+[![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
+[![Build Status](https://api.travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
+
+Przedstawiona w tym dokumencie jest lista dla początkujących w kwestii wnoszeniu własnego wkładu (kontrybucja) do tzw. Otwartego Oprogramowania *(ang. contributing to Open Source)*
+
+Jeżeli posiadasz bądź znajdziesz niewymienione w tym dokumencie źródła, prosze dodaj [pull request](https://panizkomputerem.pl/pull-request/)
+
+Jeżeli masz jakiekolwiek pytania bądź komentarz stwórz kwestię *(ang. issue)* w tym repozytorium.
+
+**Zawartość**
+
+- [Ogólne wnoszenie własnego wkładu do otwartego oprogramowania](#Ogólne-wnoszenie-własnego-wkładu-do-otwartego-oprogramowania)
+- [Bezpośrednie kategorie wyszukiwania kwestii na GitHubie](#Bezpośrednie-kategorie-wyszukiwania-kwestii-na-GitHubie)
+- [Środowisko kontrybucji Mozilli](#Środowisko-kontrybucji-Mozilli)
+- [Użyteczne artykuły dla początkujących w kontrybuowaniu do otwartego oprogramowania](#Użyteczne-artykuły-dla-początkujących-w-kontrybuowaniu-do-otwartego-oprogramowania)
+- [Używanie kontroli wersji](#Używanie-kontroli-wersji)
+- [Książki o otwartym oprogramowaniu](#Książki-o-otwartym-oprogramowaniu)
+- [Inicjatywy kontrybucji w otwartym oprogramowaniu](#Inicjatywy-kontrybucji-w-otwartym-oprogramowaniu)
+- [Licencja](#Licencja)
+
+## Ogólne wnoszenie własnego wkładu do otwartego oprogramowania
+- [Poradnik Open Source](http://opensource.guide/pl/how-to-contribute/) **[PL]** - Strona w języku polskim poświęcona projektom otwartego oprogramowania.
+- [The Definitive Guide to Contributing to Open Source](https://www.freecodecamp.org/news/the-definitive-guide-to-contributing-to-open-source-900d5f9f2282/) by [@DoomHammerNG](https://twitter.com/DoomHammerNG)
+- [An Intro to Open Source](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) - Poradnik stworzony przez DigitalOcean by poprowadzić cię do sukcesu w kontrybuowaniu na GitHubie.
+- [Issuehub.io](http://issuehub.io/) - narzędzie do wyszukiwania kwestii na podstawie etykiet *(ang. Labels)* i języków.
+- [Code Triage](https://www.codetriage.com/) - kolejne bardzo fajne narzędzie do znajdowania popularnych repozytoriów i filtrowania kwestii na podstawie języka.
+- [Awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - GitHub repozytorium które gromadzi projekty z dobrymi bugami dla początkujących kontrybutorów i stosuje etykiety do ich opisywania.
+- [Open Source Guides](https://opensource.guide/) - Zbiór poradników dla pojedyńczych, grup i firm które chcą się nauczyć jak kontrybuować do projektów otwartego oprogramowania.
```

Comment:
"dla pojedyńczych, grup i firm które" -> "dla wszystkich, którzy"
Loan translation. Btw, "pojedyńczych" is misspelled.
"nauczyć jak" -> "nauczyć, jak"
Punctuation mistake.